homeassistant/components/climacell/const.py

```diff
@@ -246,6 +246,8 @@ def __post_init__(self) -> None:
         key=CC_ATTR_PRECIPITATION_TYPE,
         name="Precipitation Type",
         value_map=PrecipitationType,
+        device_class="climacell__precipitation_type",
```

Comment:
Are you sure this is the right way to reference those translations? I don't think it is allowed to use a non existing device_class.

(however I don't know the actual fix myself, I am looking for that as well).